### ISSUE TITLE
zero timeout

### ISSUE THREAD

**Issue by alexryabtsev** [sebleier/django-redis-cache#136]
Hi. I've found some inconsistency in documents.

The documentation [says](https://github.com/sebleier/django-redis-cache/blame/master/docs/api.rst#L31) that timeout with `None` value cache is set indefinitely. And according to the implementation of [`BaseRedisCache._set`](https://github.com/sebleier/django-redis-cache/blob/master/redis_cache/backends/base.py#L262) method it also works for zero value.

On the other hand, [official Django documentation](https://docs.djangoproject.com/en/1.11/topics/cache/#basic-usage) says that: _A timeout of 0 won’t cache the value._

Please let me know what do you think about this.

**Comment by ZedNaught** [comment]
I agree it's quite confusing to directly contradict the Django docs on this. If you don't intend to change the behavior, there should at least be a very visible notice in the docs or readme.

**Comment by bogdanpetrea** [comment]
Combine this with the automatic casting of the timeout to int, and you'll have any timeout between 0 and 1 last indefinitely as well. Wasted a couple of hours because of this.
